Add Swift imports to descriptions

shepazon opened the pull request:
This PR adds `import AWSfoo` to each Swift record in the metadata for examples that currently exist. This causes the correct import for the function(s) being used to be included in context as a convenience to the reader.

---
_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

Laren-AWS reviewed (dismissed):
I'm not very keen on this. In general, we should use as little ZonBook XML as possible in the descriptions. It would be much better to wrap the import statement in `snippet-start/-end` tags in the code files and add them as an additional tag in the list, like:

```
snippet-tags:
  - cognitoidentity.swift.import
  - cognitoidentity.swift.create-identity-pool
```

shepazon commented:
Yeah, I was afraid that would be the opinion. I'll redo this.

shepazon commented:
Still not ready... pushed just part of the work. Rest to come soon...

shepazon commented:
This should be clean to review now. I updated the writeme config file for Swift (though its output for Cognito Identity is wrong without hand-massaging afterward; see below). Also, I've added snippet tags to create a `_service-name_.swift.import` tag that can be used to insert the lines of Swift `import` statements needed for the example to work. Any example that needs additional imports have snippet tags like `_service-name_-_function-name_.swift.import`.

Cognito Identity output is not correct after generating it with `writeme.py` because the `name` for Cognito Identity is `cognito-identity` but the name used for links in the reference needs to be `cognitoidentity`. There's not any apparent way to override this. For now, I have hand-edited the generated `README.md` file to get this PR to hopefully move forward so I can progress with doc work as we barrel toward the September 17 GA date for the AWS SDK for Swift.

shepazon commented:
Ready for review again, hopefully. :)
